job-service-db/src/main/resources/procedures/getJobs.sql

```diff
@@ -134,6 +141,10 @@ BEGIN
         END IF;
     END IF;
 
+    IF in_filter IS NOT NULL THEN
+        sql := sql || whereOrAnd || in_filter;
```

Comment:
Don't you need brackets to ensure that OR expressions are handled correctly?  (Perhaps the condition object is already handling that in the Java?)

```suggestion
        sql := sql || whereOrAnd || '(' || in_filter || ')';
```